nntrainer/graph/network_graph.cpp

```diff
@@ -796,7 +713,8 @@ setInplaceSharedMemoryConfigByLayer(const std::shared_ptr<LayerNode> &lnode,
     shared_var = true;
     shared_grad = true;
   }
-  /** @todo for addition layer, variables are not shared but gradients are */
+  /** @todo for addition layer, variables are not shared but gradients are
```

Comment:
Could you check the indentation in clang-format?

Reply:
This comment needed to be deleted. I've removed it. Thank you!